cli/azd/pkg/input/console.go

```diff
@@ -102,18 +103,19 @@ func (c *AskerConsole) Confirm(ctx context.Context, options ConsoleOptions) (boo
 
 // PromptTemplate ask the user to select a template.
 // An empty string is returned if the user selects 'Empty Template' from the choices
-func (c *AskerConsole) PromptTemplate(ctx context.Context, message string) (string, error) {
+func (c *AskerConsole) PromptTemplate(ctx context.Context, message string) (templates.Template, error) {
+	var result templates.Template
```

Comment:
Seems like we could remove this and return `nil` explicitly instead in error case.

Reply:
Going to hold off on changing the contract and keep as `(Template, error)`